### DISCUSSION TITLE
Are LINKS with protocol:// or https:// are supposed to miss a "/"?

### DISCUSSION THREAD

**Discussion by PatrykWalach** [jycouet/kitql#510] [Q&A] [answered]
I've noticed my LINKS with https:// are not generated without one /.
To me, this does not seem like an expected behavior.

Is this intentional?

Inside the repo I've found tests that describe this behavior.
 
in: [plugins.spec.ts:264](https://github.com/jycouet/kitql/blob/59fbc95a08c4d115c730d393fb324bbf031f65cf/packages/vite-plugin-kit-routes/src/lib/plugins.spec.ts#L264C18-L264C44)
```ts
LINKS: {
      // reference to a hardcoded link
      twitter: 'https://twitter.com/jycouet',
```
out: [plugins.spec.ts:523](https://github.com/jycouet/kitql/blob/59fbc95a08c4d115c730d393fb324bbf031f65cf/packages/vite-plugin-kit-routes/src/lib/plugins.spec.ts#L523C16-L523C16)
```ts
    {
      name: 'direct link',
      kind: 'LINKS',
      results: 'https:/twitter.com/jycouet',
      key_path: 'twitter',
      key_symbol: 'twitter',
      params: [],
      params_shortened: [],
    }
```

**Comment by jycouet** [comment]
Great catch, seems something is wrong, I'll look asap 👍

**Comment by jycouet** [comment, marked as answer]
@PatrykWalach it should be fixed in `vite-plugin-kit-routes@0.2.0-next.4` 🚀
Let me know
